mesonbuild/modules/gnome.py

```diff
@@ -37,7 +37,7 @@
 #
 # https://github.com/ninja-build/ninja/issues/1184
 # https://bugzilla.gnome.org/show_bug.cgi?id=774368
-gresource_dep_needed_version = '>= 2.52.0'
+gresource_dep_needed_version = '>= 2.51.2'
```

Comment:
Turns out 2.51.1 has it also. Though honestly I don't think this matters *that* much, its not like anybody is releasing software depending on a dev release.